box/config.yml

```diff
@@ -97,6 +96,7 @@ php_xdebug_idekey: PHPSTORM
 php_xdebug_max_nesting_level: 256
 php_xdebug_remote_port: "9000"
 php_memory_limit: "512M"
+php_sendmail_path: "/usr/sbin/sendmail -t -i"
```

Comment:
I wanted to have real email sending and receiving it in gmail for testing.

By default VM uses mailhog and puts emails there. I wasnt aware of the problem with mailgun and having only one email listed to go through so I was looking for solution and thought maybe mailhog was the problem why my emails are not going out. Turns out that wasnt problem, this is not needed since mailgun is "overriding" mailhog anyway and it doesnt matter.